code/pluginbuild.xml

```diff
@@ -6229,6 +6229,13 @@ jar-all-plugins - Generate the plugin jar files
 				</patternset>
 			</fileset>
 		</jar>
+		<jar jarfile="${systemlstplugins.dir}/GameMode-CodeControlToken-STATSCORE.jar" manifest="${src.java.dir}/plugin/lsttokens/manifest.mf">
```

Comment:
is this auto-generated? If not, can we make it so? Its hard to review and seems to be largely redundant.